data-prepper-script/build.gradle

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+plugins {
+    id 'antlr'
+    id 'java'
+    id 'idea'
+}
+
+group 'org.opensearch.dataprepper'
+version '1.3.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    antlr 'org.antlr:antlr4:4.9.2'
+    implementation project(':data-prepper-api')
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.13.1'
```

Comment:
Jackson versions are defined by the parent project. You should remove the version from this `implementation` to use the parent version.